### PR TITLE
Bugfix/remove wc ready event

### PR DESCRIPTION
**Pull request by AndyOGo**
Fixes # (issue).

**Do NOT** merge before #507 

Changes proposed in this pull request:

 - do we need the `wcdomready` event
 - check https://github.com/webcomponents/custom-elements#customelementspolyfillwrapflushcallback
 -

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
 
 # How Has This Been Tested?
 
 
 
 # Checklist:
 
 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AndyOGo** [comment]
React Demo page:
`Uncaught DOMException: Failed to execute 'define' on 'CustomElementRegistry': this name has already been used with this registry`

**Comment by AndyOGo** [comment]
Fixed the `DomException` by avoiding multiple calls to `customElements.define`

**Comment by AndyOGo** [comment]
all errors should be fixed yeah

File: src/js/define-once.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* globals customElements */
+
+export default function defineOnce(tagName, ctor) {
```
Comment (LucaMele) [inline src/js/define-once.js]: nice

**Comment by AndyOGo** [comment]
`[object Error]{message: “Can’t call ...“, name: “TypeError”, stack: “TypeError: ...“}`

virtualbox_ie11 - win7_05_07_2018_17_24_34

**Comment by AndyOGo** [comment]
**Header-Meta-Right** uses `Array.from` and is not rendered at IE11:

virtualbox_ie11 - win7_05_07_2018_17_30_51

File: src/components/m-header-meta-right/_template.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import html from 'nanohtml';
 
-export default (props, { children }) => html`
+// IE11 does not support children on fragments
+// ref: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10060579/
+export default (props, { children, childNodes }) => html`
   <ul class="m-header-meta-right__list">
-    ${children && Array.from(children).map(child => html`
+    ${Array.from(children || childNodes).map(child => html`
```
Comment (LucaMele) [inline src/components/m-header-meta-right/_template.js]: This logic should be in Abstarct not inside the template

Comment (AndyOGo) [reply src/components/m-header-meta-right/_template.js]: done

**Comment by AndyOGo** [comment]
IE 11: Logo lost and how to restore it:

ie-restore-logo-lost

**Comment by LucaMele** [comment]
IM on it

**Pull request merged**